front/public/locales/en/translation.json

```diff
@@ -5,6 +5,7 @@
     "back": "Back",
     "cancel": "Cancel",
     "change": "Change",
+    "choose": "choose…",
```

Comment:
```suggestion
    "choose": "Choose…",
```

Reply:
for uniformity, I think it should be capitalized.

Reply:
See comment below

Reply:
Ok perfect, I didn't know that !